### PR TITLE
ST-1812: Storefront migration to .NET 6

### PR DESCRIPTION
**Pull request by Flanker72**
Description: 
Closes ST-1218. Migrating vc-storefront to .NET6, upgrading dependencies.

--

QA-test: ST-1954

Demo-test: ST-1955

Download artifact URL: ghcr.io/virtocommerce/storefront:6.0.0-pr-602-dotnet-6-migration-5d6ae016

### CODE REVIEW

File: VirtoCommerce.LiquidThemeEngine/ShopifyLiquidThemeEngine.cs [hunk 1]
```diff
@@ -364,7 +364,7 @@ public ValueTask<string> RenderTemplateAsync(string templateContent, string temp
                 NewLine = Environment.NewLine,
                 TemplateLoaderLexerOptions = new LexerOptions
                 {
-                    Mode = ScriptMode.Liquid
+                    Mode = ScriptMode.Default
```
Comment (tatarincev) [inline VirtoCommerce.LiquidThemeEngine/ShopifyLiquidThemeEngine.cs:367]: I'm afraid this mode will broke exists liquid templates if they are still exists. Why did you change it to default?

Comment (Vectorfield4) [inline VirtoCommerce.LiquidThemeEngine/ShopifyLiquidThemeEngine.cs:367]: Why?

**Review by Vectorfield4 — DISMISSED** [review]

File: VirtoCommerce.LiquidThemeEngine/ShopifyLiquidThemeEngine.cs (hunk 1 above)
Comment (Flanker72) [reply VirtoCommerce.LiquidThemeEngine/ShopifyLiquidThemeEngine.cs:367]: image
Seems that Liquid doesn't exists now - it causes build error.

**Review by tatarincev — APPROVED** [review]
